docs/section5/Rule5-43.md

```diff
@@ -0,0 +1,36 @@
+
+# Envelope - Rule 5-43  
+
+**Rule ID:** 5-43  
+**Rule Description:** Automatic fenestration shading devices shall not be modeled in the Baseline.  
+**Rule Assertion:** Baseline RMD = expected value  
+**Appendix G Section:** Section G3.1-5(d) Building Envelope Modeling Requirements for the Baseline building  
```

Comment:
I feel like I am going crazy, I don't see where it addresses automatic shading devices for the baseline in the Baseline Building Performance column in d.

Reply:
this is ASHRAE 2022

Reply:
Still in 90.1 2022 not seeing anything about shading in Section G3.1-5(d) Building Envelope Modeling Requirements for the Baseline building

Reply:
it's actually 5f, but it's there.  I updated the reference:

![image](https://github.com/user-attachments/assets/a4d16c26-966f-45bd-8e13-82c76572caa2)